src/entity/View.ts

```diff
@@ -21,7 +21,10 @@ export class View {
   @Column({ type: 'text', nullable: true })
   referer?: string;
 
-  @PrimaryColumn({ default: () => 'now()' })
+  @PrimaryColumn({ default: () => 'now()', precision: 3 })
```

Comment:
Why do we need this precision?

Reply:
Oh, sorry I will elaborate further the current description in the Docs Page.

Most languages follow the [ISO format](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) in date which is 3 digits maximum for milliseconds, and when we store 6 (default) it won't match due to the missing additional 3 digits of milliseconds in the database.

Reply:
But do we actually need a ms level precision? I'm not sure

Reply:
Good point. We can then set this to `precision: 0` since if we compare a date vs a timestamp with `000` milliseconds (appended by most formatter as shown below) it would still provide the expected result.

![Screen Shot 2021-11-11 at 7 15 25 AM](https://user-images.githubusercontent.com/13744167/141208798-4a0264a5-8acb-4189-a7d9-3ee8afd38c57.png)